tests/ClientTest.php

```diff
@@ -64,6 +64,42 @@ public function registerShutdownFunction()
 
 class ClientTest extends TestCase
 {
+    /**
+     * @dataProvider constructorInitializesTransactionStackDataProvider
```

Comment:
I would prefer to split this test in two instead of using this strange data provider